src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat[3];
+  const toSeparator = toFormat[3];
+  const dateArr = date.split(fromSeparator);
+  const dateObj = {};
+  const newDateArr = [];
+
+  fromFormat.pop();
+  toFormat.pop();
```

Comment:
The `pop()` method removes the last element from an array and returns that element, so that in this way the `fromSeparator` and `toSeparator` variables can be immediately defined.